numpy/core/fromnumeric.py

```diff
@@ -1814,6 +1814,10 @@ def ravel(a, order='C'):
 
     Notes
     -----
+    It reads a 1-D flattened array as ``a``. The returned value ``y`` is a
+    contiguous 1-D array of the same subtype as ``a``,
+    with shape ``(a.size,)``.
+
```

Comment:
I still think this can be collapsed into the `Returns` directive above (line 1804). Do you disagree?

Reply:
ohhh I'm sorry I misunderstood your point.  I certainly agree!